libcontainer/cgroups/fs2/fs2.go

```diff
@@ -233,21 +233,26 @@ func (m *Manager) setUnified(res map[string]string) error {
 		if strings.Contains(k, "/") {
 			return fmt.Errorf("unified resource %q must be a file name (no slashes)", k)
 		}
-		if err := cgroups.WriteFile(m.dirPath, k, v); err != nil {
-			// Check for both EPERM and ENOENT since O_CREAT is used by WriteFile.
-			if errors.Is(err, os.ErrPermission) || errors.Is(err, os.ErrNotExist) {
-				// Check if a controller is available,
-				// to give more specific error if not.
-				sk := strings.SplitN(k, ".", 2)
-				if len(sk) != 2 {
-					return fmt.Errorf("unified resource %q must be in the form CONTROLLER.PARAMETER", k)
-				}
-				c := sk[0]
-				if _, ok := m.controllers[c]; !ok && c != "cgroup" {
-					return fmt.Errorf("unified resource %q can't be set: controller %q not available", k, c)
+		// To support write multiple item into one cgroup file like:
+		// {"io.max": "252:1 riops=1000 wiops=1000 \n 252:2 riops=2000 wiops=2000"}
+		items := strings.Split(v, "\n")
+		for _, item := range items {
+			if err := cgroups.WriteFile(m.dirPath, k, item); err != nil {
```

Comment:
Just curious, why is this writing two lines instead of overwriting them?

If you had a test case for this, I could have answered myself by looking at that ;)

Reply:
the cgroup file is also like a map. if the key is the same, the new one will override old one, but if it is different key, like different device number, the will add a new line

Reply:
Not sure how to add test case at here, there is the test by my environment
```
root@tess-node-6bg2h:/sys/fs/cgroup/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pod02d6c4d8_0888_464a_9037_7238679d4174.slice# cat io.max 
252:0 rbps=1048576 wbps=1048576 riops=100 wiops=100
253:0 rbps=1048576 wbps=1048576 riops=100 wiops=100
root@tess-node-6bg2h:/sys/fs/cgroup/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pod02d6c4d8_0888_464a_9037_7238679d4174.slice# echo  "252:16 rbps=1048576 wbps=1048576 riops=100 wiops=100" > io.max 
root@tess-node-6bg2h:/sys/fs/cgroup/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pod02d6c4d8_0888_464a_9037_7238679d4174.slice# cat io.max 
252:16 rbps=1048576 wbps=1048576 riops=100 wiops=100
252:0 rbps=1048576 wbps=1048576 riops=100 wiops=100
253:0 rbps=1048576 wbps=1048576 riops=100 wiops=100
```

Reply:
I added test case in pr @rata